tests/controller-test.js

```diff
@@ -242,6 +242,7 @@ OO = {
     Html5Skin.onPlayerCreated.call(controllerMock, 'customerUi', elementId, {skin:{config:{}}}, persistentSettings);
     Html5Skin.onSkinMetaDataFetched.call(controllerMock, 'customerUi', {});
     Html5Skin.loadConfigData.call(controllerMock, 'customerUi', {skin:{config:{}}}, {}, {}, {});
```

Comment:
Do we need tests for skin:{config:[]} as well?  And {inline:{}}, just in case. Or do those already exist.